src/test/int/moveToHeading.int.test.ts

```diff
@@ -0,0 +1,208 @@
+import { virtual } from "../..";
+
+const headingLevels = ['1', '2', '3', '4', '5', '6'];
+
+describe("Move To Heading", () => {
+  afterEach(async () => {
+    await virtual.stop();
+    document.body.innerHTML = "";
+  });
+
+  describe("moveToNextHeading", () => {
+    describe("when there is a level 1 heading in the container", () => {
+      beforeEach(async () => {
+        document.body.innerHTML = `
+          <h1 aria-label="Accessible name">Heading</h1>
```

Comment:
```suggestion
          <h1>Heading</h1>
```

It is typically an anti-pattern to include aria based labelling where the content is sufficient.

With the tests the hope is that they can somewhat demonstrate best practices - if we want to test that a label can be used here then it can be an extra test case rather than the default for all tests.

Reply:
Same applies to subsequent tests

Reply:
I've put some tests for `<div role="heading" aria-level="7">` in this test file. I can shift it to `moveToRole.int.test.ts` if you like -- I just didn't read this comment properly!

I used `aria-level="7"` because I've heard one legitimate use of explicit `role="heading"` is for legal documents where heading levels can go deeper than the standard six.

Reply:
Yep, good to support `aria-level` past 6, see https://a11ysupport.io/tech/aria/aria-level_attribute